TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java

```diff
@@ -432,36 +432,36 @@ public static void loadmaps(CommandContext cmd, CommandSender sender) {
     @Command(aliases = {"channel", "chatchannel", "cc"}, desc = "Change or select a chat channel.", usage = "(all|team|staff)", min = 1)
     public static void channel(CommandContext cmd, CommandSender sender) {
         if(!(sender instanceof Player)) {
-            sender.sendMessage("Error: Only players can use this command.");
+            sender.sendMessage("Only players can use this command.");
             return;
         }
         Player player = (Player) sender;
         if (cmd.argsLength() == 0) {
-          player.sendMessage(ColorConverter.filterString("&cUsage: /channel (name)"));
+          player.sendMessage(ChatColor.RED + "Usage: /channel (name)");
           return;
         }
 
         String channelName = cmd.getString(0).toUpperCase();
         ChatModule.Channel channel = ChatModule.Channel.byName(channelName);
         if (channel == null) {
-          player.sendMessage(ColorConverter.filterString("&cInvalid channel: " + channelName));
+          player.sendMessage(ChatColor.RED + "Invalid channel: " + channelName);
           player.sendMessage(ColorConverter.filterString("&cChannels: ( " + StringUtils.join(Arrays.stream(ChatModule.Channel.values()).filter(ch -> ch.hasPermission(player)).collect(Collectors.toList()), " | ")) + " )");
           return;
         }
 
         if (!channel.hasPermission(player)) {
-          player.sendMessage(ColorConverter.filterString("&cError: Insufficient permissions."));
+          player.sendMessage(ChatColor.RED + "Insufficient permissions.");
           return;
         }
 
         ChatModule.getChannels().put(player.getUniqueId().toString(), channel);
-        player.sendMessage(ColorConverter.filterString("&7You've been added to the channel &c&l" + channel.name() + "&7."));
+        player.sendMessage(ChatColor.GRAY + "You are now talking in " + ChatColor.BOLD.toString() + ChatColor.RED + channel.name());
     }
 
     @Command(aliases = {"t"}, desc = "Send a message to your team.", usage = "(message)", min = 1)
     public static void t(CommandContext cmd, CommandSender sender) {
         if (!(sender instanceof Player)) {
-            sender.sendMessage(ChatColor.RED + "You must be a player to do that.");
+            sender.sendMessage("Only players can use this command.");
```

Comment:
Does this not make it send a white message instead of what is already there?

Reply:
Console doesnt have colors as far as I know.

Reply:
Console does have colors think thonk